app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -66,6 +66,7 @@ def course_json
         provider_ucas_code: application_choice.provider.code,
         site_ucas_code: application_choice.site.code,
         course_ucas_code: application_choice.course.code,
+        level: application_choice.course.level,
```

Comment:
Do we need to add this? I think it's only for our internal use so far

Reply:
Removed for now.